db/db.go

```diff
@@ -227,12 +231,19 @@ func (db *DB) Query(queries []string, tx, xTime bool) ([]*Rows, error) {
 			}
 		}()
 
+		if db.sqlite3conn == nil {
```

Comment:
I don't think this will work because another call to `Open` could take place and set a new value for `sqlite3conn`. That said, I'm going to fix this connection handling in another PR so the issue will soon be moot.